nbbuild/external/binaries-list

```diff
@@ -14,7 +14,7 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-A7674A6D78B7FEA58AF76B357DAE6EA5E3FDFBE9 apitest-70.jar
 16398550402B27F81CD0D508CEF54B3E47A4A6DA org.apache.rat:apache-rat:0.12
 AB396EE119BFAD809CC9F09950CC22E2BCE2FE35 langtools-9.zip
 E32B3483FBF362C92088CB79E9F1F161F3F64A21 org.apidesign.javadoc:codesnippet-doclet:0.30
+D5CC2CD2A20963B86CF95397784BC7E74101C7A9 org.netbeans.tools:sigtest-maven-plugin:1.2
```

Comment:
Please remove the `nbbuild/external/apitest-70-license.txt`, as the corresponding binary is not used anymore. For the `sigtest-maven-plugin-1.2.jar` please add a license file.

Reply:
Once branch was synced with  apache::master issue in nbbuild/external  got resolved